README.md

```diff
@@ -48,18 +48,18 @@ With development builds, internal NuGet feeds are necessary for some scenarios (
+| **Linux-musl** | [![][linux-musl-badge-master]][linux-musl-version-master]<br>[tar.gz][linux-musl-targz-master] - [Checksum][linux-musl-targz-checksum-master] | [![][linux-musl-badge-5.0.1XX-preview4]][linux-musl-version-5.0.1XX-preview4]<br>[tar.gz][linux-musl-targz-5.0.1XX-preview4] - [Checksum][linux-musl-targz-checksum-5.0.1XX-preview4] | [![][linux-musl-badge-5.0.1XX-preview3]][linux-musl-version-5.0.1XX-preview3]<br>[tar.gz][linux-musl-targz-5.0.1XX-preview3] - [Checksum][linux-musl-targz-checksum-5.0.1XX-preview3] | [![][linux-musl-badge-3.1.3XX]][linux-musl-version-3.1.3XX]<br>[tar.gz][linux-musl-targz-3.1.3XX] - [Checksum][linux-musl-targz-checksum-3.1.3XX] | [![][linux-musl-badge-3.1.2XX]][linux-musl-version-3.1.2XX]<br>[tar.gz][linux-musl-targz-3.1.2XX] - [Checksum][linux-musl-targz-checksum-3.1.2XX] | [![][linux-musl-badge-3.1.1XX]][linux-musl-version-3.1.1XX]<br>[tar.gz][linux-musl-targz-3.1.1XX] - [Checksum][linux-musl-targz-checksum-3.1.1XX] | [![][linux-musl-badge-3.0.1XX]][linux-musl-version-3.0.1XX]<br>[tar.gz][linux-musl-targz-3.0.1XX] - [Checksum][linux-musl-targz-checksum-3.0.1XX] |
+| **Windows arm** | [![][win-arm-badge-master]][win-arm-version-master]<br>[zip][win-arm-zip-master] - [Checksum][win-arm-zip-checksum-master] | [![][win-arm-badge-5.0.1XX-preview4]][win-arm-version-5.0.1XX-preview4]<br>[zip][win-arm-zip-5.0.1XX-preview4] - [Checksum][win-arm-zip-checksum-5.0.1XX-preview4] | [![][win-arm-badge-5.0.1XX-preview3]][win-arm-version-5.0.1XX-preview3]<br>[zip][win-arm-zip-5.0.1XX-preview3] - [Checksum][win-arm-zip-checksum-5.0.1XX-preview3] | [![][win-arm-badge-3.1.3XX]][win-arm-version-3.1.3XX]<br>[zip][win-arm-zip-3.1.3XX] - [Checksum][win-arm-zip-checksum-3.1.3XX] | [![][win-arm-badge-3.1.2XX]][win-arm-version-3.1.2XX]<br>[zip][win-arm-zip-3.1.2XX] - [Checksum][win-arm-zip-checksum-3.1.2XX] | [![][win-arm-badge-3.1.1XX]][win-arm-version-3.1.1XX]<br>[zip][win-arm-zip-3.1.1XX] - [Checksum][win-arm-zip-checksum-3.1.1XX] | [![][win-arm-badge-3.0.1XX]][win-arm-version-3.0.1XX]<br>[zip][win-arm-zip-3.0.1XX] - [Checksum][win-arm-zip-checksum-3.0.1XX] |
+| **Windows arm64** | [![][win-arm64-badge-master]][win-arm64-version-master]<br>[zip][win-arm64-zip-master] - [Checksum][win-arm64-zip-checksum-master] | [![][win-arm64-badge-5.0.1XX-preview4]][win-arm64-version-5.0.1XX-preview4]<br>[zip][win-arm64-zip-5.0.1XX-preview4] - [Checksum][win-arm64-zip-checksum-5.0.1XX-preview4] | [![][win-arm64-badge-5.0.1XX-preview3]][win-arm64-version-5.0.1XX-preview3]<br>[zip][win-arm64-zip-5.0.1XX-preview3] | **N/A** | **N/A** | **N/A** | **N/A** |
```

Comment:
checksum is missing for preview3 column

Reply:
Good catch. It wasn't there for p2 and I basically just renamed p3->p4 and p2->p3